seccomp.go

```diff
@@ -34,19 +41,27 @@ type VersionError struct {
 	minimum string
 }
 
+// Caches the libseccomp API level
+var apiLevel uint
+
+func init() {
+	// This forces the cgo libseccomp to initialize its internal API support state,
+	// which is necessary in order to other seccomp APIs to work correctly.
+	GetAPI()
```

Comment:
That isn't the intention and I *think* we resolved this in the latest v2.5.x release, however I will concede that this is likely necessary for older versions of libseccomp.  Perhaps update the comment to mention this is necessary on *older* versions of libseccomp?